addons/account/controllers/portal.py

```diff
@@ -46,7 +46,7 @@ def portal_my_invoices(self, page=1, date_begin=None, date_end=None, sortby=None
             sortby = 'date'
         order = searchbar_sortings[sortby]['order']
 
-        archive_groups = self._get_archive_groups('account.move', domain)
+        archive_groups = self._get_archive_groups('account.move', domain) if values.get('my_details') else []
```

Comment:
@tde-banana-odoo The `archive_groups` feature of the portal isn't shown anywhere anymore but its values are still computed on portal pages.
Can we totally remove the feature or do you expect a comeback of the feature someday ? :thinking:

Reply:
I don't even know what this is about. Even if we want to make this unknown feature back someday, no need to keep dead code. If it is not used let us remove it.